module-code/app/com/codacy/play/tomcatcp/pool/TomcatCPPool.java

```diff
@@ -29,7 +29,7 @@ public void setMetricRegistry(MetricRegistry metricRegistry) {
 
     public void setMetricsTrackerFactory(MetricsTrackerFactory metricsTrackerFactory) {
         this.metricsTracker = Optional.of(
-                metricsTrackerFactory.create("tomcatcp." + databaseName, getPoolStats())
+                metricsTrackerFactory.create("tomcatcp." + databaseName + "." + getName(), getPoolStats())
```

Comment:
We should take care of the null case. This was if it is not present will have an unexpected outcome.